lib/facilities/drivetime_bands/client.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+require 'common/client/base'
+
+module Facilities
+  module DrivetimeBands
+    class DrivetimeBandDownloadError < StandardError
+    end
+
+    # Core class responsible for api interface operations
+    class Client < Common::Client::Base
+      configuration Facilities::DrivetimeBands::Configuration
+
+      def get_drivetime_bands(offset, limit)
+        params = build_params(offset, limit)
+        response = perform(:get, '/arcgis2/rest/services/Portal/MonthlyVAST_TTB/FeatureServer/0/query', params)
+        drivetime_band_response = Facilities::DrivetimeBands::Response.new(response.body)
+        response_body = drivetime_band_response.parsed_json
```

Comment:
This line is unnecessary now

Reply:
response_body = drivetime_band_response.parsed_json is referring to an instance variable which now equals the parsed json response body

Reply:
Instead of reassigniing `drivetime_band_response.parsed_json` to another instance variable, you could use `drivetime_band_response.parsed_json` in the error checking step. I think this would make the code more cohesive, since `drivetime_band_response` is used again later in the method

Reply:
Ah, I see what you're saying now 😄